protocol/msgpack/aggregated_encoder.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/m3db/m3metrics/policy"
 )
 
-type encodeRawMetricWithPolicyFn func(data []byte, p policy.Policy)
+type encodeRawMetricWithPolicyFn func(data []byte, p policy.StoragePolicy)
```

Comment:
`encodeRawMetricWithStoragePolicyFn`? Can we do a sweep of the code base and replace policy with storagePolicy in function names / comments etc where applicable?